.nycrc

```diff
@@ -4,13 +4,17 @@
     "text-summary"
   ],
   "check-coverage": true,
-  "statements": 87,
-  "branches": 75,
-  "functions": 82,
-  "lines": 88,
-  "exclude": [
-    "test/**/*.js",
-    "apps/**/sections/**/*.js",
-    "apps/**/fields/**/*.js"
-  ]
+  "coverageThreshold": {
+    "*/**": {
+      "statements": 87,
+      "branches": 75,
+      "functions": 82,
+      "lines": 88,
+      "exclude": [
+        "test/**/*.js",
+        "apps/**/sections/**/*.js",
+        "apps/**/fields/**/*.js"
+      ]
+    }
+  }
```

Comment:
What's the aim of setting this? I see teams use it as part of their CI/CD

Reply:
It checks that test coverage has met the threshold for each file (minus those in the exclude list)

Reply:
Yeah I remember using it in another team.  We set it up and it would block WIP PRs from being pushed to git unless you have the `--no-verify` tag.  I'm guessing in this instance it won't block a dev from pushing?
My question is what made you want to put it? Why did you pick those values?

Reply:
Those values were already set previously but were set globally (so it it took the average value of all files).  I added in them into the coverageThreshold property and set the files to "*/**" because if it failed globally it would stop the test script running even when all the tests were passing. Using the coverageThreshold makes it check the values against files individually and therefore doesn't break the test if all the tests are passing but the global average is slightly less.

Reply:
It was already there and you are improving it, that makes sense, thanks for explaining